simpledotorg/simple-server
# Update text and descriptions for the DM dashboard

**Story card:** [sc-13414](https://app.shortcut.com/simpledotorg/story/13414/ui-feedback)

## Because

Texts in the DM dashboards needed some minor changes.

## This addresses

Updating the texts as mentioned in the story.

## Test instructions

Start localhost server and load diabetes dashboard, it should reflect the changes as mentioned in the story.
